module/VuFind/src/VuFind/RecordDriver/AbstractBase.php

```diff
@@ -94,6 +88,7 @@ public function __construct($mainConfig = null, $recordConfig = null)
     {
         $this->mainConfig = $mainConfig;
         $this->recordConfig = $recordConfig ?? $mainConfig;
+        $this->setSourceIdentifiers('Solr');
```

Comment:
Yes. Done. Another possibility would be make the default empty or something so that it's not accidentally left as is.

Reply:
If we default the properties to empty strings in the trait, can we drop this set call entirely? It does seem like maybe it would be better to fail due to an empty string than to accidentally refer to the wrong backend if something is overlooked.

Reply:
Yes, and now done. Mink tests are still passing. :)